packages/commonwealth/server/migrations/20241008123832-add-token-schema-updates.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(async (t) => {
+      await queryInterface.addColumn(
+        'Tokens',
+        'community_id',
+        {
+          type: Sequelize.STRING,
+          allowNull: false,
+          references: { model: 'Communities', key: 'id' },
+          onUpdate: 'CASCADE',
+          onDelete: 'SET NULL',
+        },
+        { transaction: t },
+      );
+
+      await queryInterface.addConstraint('Tokens', {
+        type: 'foreign key',
+        fields: ['community_id'],
+        name: 'fk_Tokens_community_id',
+        references: {
+          table: 'Communities',
+          fields: ['id'],
+        },
+        onUpdate: 'CASCADE',
+        onDelete: 'SET NULL',
+        transaction: t,
+      });
```

Comment:
This is redundant since `references` on `addColumn` will add the fk,

Reply:
done